tvtk/tvtk_base.py

```diff
@@ -168,11 +186,13 @@ class TraitRevPrefixMap(traits.TraitPrefixMap):
     keys map to the same value, one of the valid keys will be used.
 
     """
-    def __init__(self, map):
-        traits.TraitPrefixMap.__init__(self, map)
+    def __init__(self, map, *extra_values, **kwargs):
+        super().__init__(map, **kwargs)
         self._rmap = {}
         for key, value in map.items():
             self._rmap[value] = key
+        for key in extra_values:
+            self._rmap[key] = kwargs['default_value']
```

Comment:
This I am not sure about. I wasn't really sure how `Trait` handled the extra values that used to be given (only used in `wrapper_gen.py` below)